src/SFA.DAS.AANHub.Api.UnitTests/Controllers/CalendarEventsControllerTests/CalendarEventsControllerPostTests.cs

```diff
@@ -20,7 +21,7 @@ public async Task CreateCalendarEvent_InvokesCommand(Guid requestedByMemberId, C
     {
         Mock<IMediator> mediatorMock = new();
         mediatorMock.Setup(m => m.Send(It.IsAny<CreateCalendarEventCommand>(), cancellationToken)).ReturnsAsync(new ValidatedResponse<CreateCalendarEventCommandResult>(new CreateCalendarEventCommandResult(requestedByMemberId)));
-        CalendarEventsController sut = new(Mock.Of<ILogger<CalendarEventsController>>(), mediatorMock.Object);
+        CalendarEventsController sut = new(Mock.Of<ILogger<CalendarEventsController>>(), mediatorMock.Object, Mock.Of<ICalendarEventsReadRepository>());
```

Comment:
Could the mock not be passed as a frozen parameter;
[Frozen] Mock calendarEventsReadRepository

Applies to other places as well